.eslintrc.json

```diff
@@ -3,7 +3,7 @@
     "browser": true
   },
   "globals": {
-    "ActiveXObject": true,
```

Comment:
If you know what is this, you are probably 35+

Reply:
confirming. almost 36 and I'm aware of it :)